content/best_practices.adoc

```diff
@@ -0,0 +1,113 @@
+= Coding best practices
+
+== Guidelines
+
+The developer and reviewer of a Pull Request must check the compliance with following guidelines documents:
+
+* https://docs.google.com/spreadsheets/d/1s-op62uot1dy4YwMA-JswxNg0DwS4sdmg1eAll0VHIU[Development Best Practices]
```

Comment:
We should move it to tribe.

Reply:
Right! moved to OnlyOffice Documents in Tribe